src/Components/Alert/AlertProvider.tsx

```diff
@@ -93,6 +93,12 @@ export const AlertProvider: FC<AlertProviderProps> = ({
     if (isEditMode || isAlertArtworksView) return
     const criteria = getAllowedSearchCriteria(initialCriteria ?? {})
 
+    // `forSale` is allowed as a filter criterion,
+    // but NOT as an alert criterion, so we remove it.
+    // (Alerts, by definition, stipulate forSale=true
+    // when they are created in Gravity.)
+    delete criteria.forSale
```

Comment:
An important detail about this filter is that we _don't_ want to pass it into the initial alert criteria.

Alerts always specify **for sale: true** as a filter criteria, which [happens upstream in Gravity by default](https://github.com/artsy/gravity/blob/702f5b020082f9264d30f89088a9c8fc7650f2c2/app/models/search_criteria.rb#L23) when the SearchCriteria gets persisted.